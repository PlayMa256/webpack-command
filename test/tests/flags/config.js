const cli = require('../../../lib');
const { prep, validate } = require('../../util');

const fixture = 'config/config';
const opts = { fixture };

describe('--config', () => {
  it(`should validate`, () => {
    expect(validate(opts)).toEqual(true);
  });

  it(`should build`, () => {
    const { argv } = prep({ fixture });

    return cli({ argv }).then((result) => {
      expect(result).toMatchSnapshot();
    });
  });
});
