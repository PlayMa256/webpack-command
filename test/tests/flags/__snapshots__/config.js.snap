// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flags > --config > should build #0 1`] = `
"Version: webpack 4.23.1
  Asset      Size  Chunks             Chunk Names
main.js  3.81 KiB    main  [emitted]  main
Entrypoint main = main.js
[./src/index.js] 43 bytes {main} [built]"
`;
